src/sentry/static/sentry/app/utils/performance/vitals/constants.ts

```diff
@@ -0,0 +1,76 @@
+import {t} from 'app/locale';
+import {measurementType, WebVital} from 'app/utils/discover/fields';
+import {Vital} from 'app/utils/performance/vitals/types';
+
+export const WEB_VITAL_DETAILS: Record<WebVital, Vital> = {
+  [WebVital.FP]: {
+    slug: 'fp',
+    name: t('First Paint'),
+    acronym: 'FP',
+    description: t(
+      'Render time of the first pixel loaded in the viewport (may overlap with FCP).'
+    ),
+    poorThreshold: 3000,
```

Comment:
Renamed from `failureThreshold` to `poorThreshold`. A TODO item here is to consolidate the duplicate thresholds defined in vital details.